source/helpers/is-format-json.js

```diff
@@ -0,0 +1,7 @@
+import minimist from 'minimist';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['minimist' should be listed in the project's dependencies. Run 'npm i -S minimist' to add it](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5102014)